Src/BootCamp.Chapter/Program.cs

```diff
@@ -6,7 +6,24 @@ class Program
     {
         static void Main(string[] args)
         {
-            Console.WriteLine("Hello World!");
+            string name;
+            int age;
+            double weight;
+            double height;
+            double BMI;
+            Console.WriteLine("BMI Calculator");
+            Console.WriteLine("Please enter your name: ");
```

Comment:
right now, code looks like a blob. Try to be more subtle and improve readability by separating unrelated code parts with a blank line.

For example, unrelated parts are:
Read age
Read name
Read weight
Calculate bmi

Reply:
Done. Thank you for the suggestions.